modules/json/json.c

```diff
@@ -348,7 +355,10 @@ json_t *get_object(pv_json_t *var, pv_param_t *pvp, json_tag **tag,
 
 int pv_get_json(struct sip_msg* msg,  pv_param_t* pvp, pv_value_t* val)
 {
-	return pv_get_json_ext(msg, pvp, val, JSON_C_TO_STRING_SPACED);
+  if (json_disable_escape_forward_slash) {
+    return pv_get_json_ext(msg, pvp, val, JSON_C_TO_STRING_SPACED | JSON_C_TO_STRING_NOSLASHESCAPE);
+  } else
+	  return pv_get_json_ext(msg, pvp, val, JSON_C_TO_STRING_SPACED);
```

Comment:
Not required, but it might be a bit cleaner to build the bitmask by checking the flag and then just have a single call to the function.

`
int flags = JSON_C_TO_STRING_SPACED;
if (json_disable_escape_forward_slash)
    flags |= JSON_C_TO_STRING_NOSLASHESCAPE;

return pv_get_json_ext(msg, pvp, val, flags);
`

Also, this new code doesn't appear to be following the indentation standard for the rest of the file.